### DISCUSSION TITLE
[minimal-blog]: Banner Image doesn't show up on the posts

### DISCUSSION THREAD

**Discussion by Ramshandilya** [LekoArts/gatsby-themes#637] [Questions] [answered]
Hi there,

Not sure if this is a bug, but I've noticed this even in the [starter template](https://minimal-blog.lekoarts.de/introduction-to-defence-against-the-dark-arts) where the [source code](https://raw.githubusercontent.com/LekoArts/gatsby-starter-minimal-blog/master/content/posts/introduction-to-defence-against-the-dark-arts/index.mdx) does have the `banner` property but the post doesn't seem to show up the banner image. 

```
---
title: Introduction to "Defence against the Dark Arts"
date: 2019-11-07
description: Defence Against the Dark Arts (abbreviated as DADA) is a subject taught at Hogwarts School of Witchcraft and Wizardry and Ilvermorny School of Witchcraft and Wizardry. In this class, students study and learn how to defend themselves against all aspects of the Dark Arts, including dark creatures, curses, hexes and jinxes (dark charms), and duelling.
tags:
  - Tutorial
  - Dark Arts
banner: ./defence-against-the-dark-arts.jpg
---
```
<img width="1011" alt="Screen Shot 2020-04-23 at 11 50 13 AM" src="https://user-images.githubusercontent.com/2031908/80137872-cb246e00-8558-11ea-93ba-ffb8eda368e4.png">

**Comment by LekoArts** [comment, marked as answer]
Hi, thanks for the issue!

The `banner` property of the frontmatter is intended for the `og:image` part of the `<head />`: https://github.com/LekoArts/gatsby-themes/blob/c30767afab1aa46edc5230673b2684d24e802e3f/themes/gatsby-theme-minimal-blog/src/components/post.tsx#L42

It gets passed to the `<SEO />` component. I didn't intend to show a image of some sort there. You could shadow https://github.com/LekoArts/gatsby-themes/blob/master/themes/gatsby-theme-minimal-blog-core/src/templates/post-query.tsx to add another query for the banner to e.g. also use it in your blog post template (you would need to shadow https://github.com/LekoArts/gatsby-themes/blob/master/themes/gatsby-theme-minimal-blog/src/components/post.tsx

Thanks for using the theme :)

**Reply by jsg2021** [reply]
I'm trying to do this... and shadowing `post-query.tsx` isn't working ...

<img width="935" alt="Screenshot 2023-04-09 at 10 15 25 AM" src="https://user-images.githubusercontent.com/577453/230781114-966c5e47-3a45-4c9a-914c-598323221503.png">

Shadowing the other queries are, and I see inverse warnings to the above:
<img width="1021" alt="Screenshot 2023-04-09 at 10 17 07 AM" src="https://user-images.githubusercontent.com/577453/230781213-1108fe46-516b-4ae8-ae92-a406fcf57ceb.png">

could the [createPage call with additional query params](https://github.com/LekoArts/gatsby-themes/blob/main/themes/gatsby-theme-minimal-blog-core/gatsby-node.mjs#L348) be breaking shadowing?

**Reply by jsg2021** [reply]
I just tested that line replaced with:

<img width="678" alt="Screenshot 2023-04-09 at 10 21 52 AM" src="https://user-images.githubusercontent.com/577453/230781450-c144d735-6825-4f12-95eb-1fb3597b9793.png">

and my shadowed file worked... however, it breaks other things. :/ I should probably find the appropriate repo for the shadow resolver and report there.

**Reply by jsg2021** [reply]
https://github.com/gatsbyjs/gatsby/issues/37925
